### DISCUSSION TITLE
/netlify-vs-vercel-vs-azure-static-web-app/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#513] [Comments]
Which service? Netlify vs Vercel vs Azure Static Web App
        
[https://www.eliostruyf.com/netlify-vs-vercel-vs-azure-static-web-app/](https://www.eliostruyf.com/netlify-vs-vercel-vs-azure-static-web-app/)

**Comment by comment-migrator** [comment]
```header
Originally posted by Lucian Ciufudean on 2021-03-11 07:32
```

Did you skip a few words here "The builds itself are very fast and allow you your site before you put it public."?

**Reply by comment-migrator** [reply]
```header
Originally posted on 2021-03-11 10:09
```

It seems words got eaten by the internet 🤣. Thanks for pointing this out!

**Comment by bendandoSolo** [comment]
Cons with Azure Static Webapps, you cannot use /api/ routing out of the box as it expects to work with azure functions
